interactive/components/Summary.vue

```diff
@@ -10,8 +13,13 @@ const isDefault = $computed(() => (userConfigRaw.value || defaultConfigRaw) ===
       <div font-bold text-right>
         {{ searcher.uno.config.presets.length }}
       </div>
-      <div op50>
+      <div op50 @mouseenter="showPresets = true" @mouseleave="showPresets = false">
         Presets
+        <div v-if="showPresets" ml-2>
+          <div v-for="item in searcher.uno.config.presets" :key="item.name">
+            {{ item.name }}
+          </div>
+        </div>
```

Comment:
This change is irrelevant, if you want see all presets, you can switch config panel.

Reply:
emm yeah I think so, but the idea came to me when I first entered the document website and want to know what presets are loaded. Perhaps that's my silliness.

So I have to revert these things?